ext/standard/tests/mail/mail_log.phpt

```diff
@@ -2,7 +2,7 @@
 Test mail() function : mail.log ini setting
 --INI--
 sendmail_path=tee /tmp/mail.out >/dev/null
-mail.log = /tmp/mail.log
+mail.log = {TMP}/mail.log
```

Comment:
Changing mail.log requires to also change the output filename of the `tee` above.

Reply:
I reverted the change, since it is not for windows.